src/tls/cockpit-certificate-helper.in

```diff
@@ -2,51 +2,88 @@
 
 set -eu
 
-# expected arguments
-[ "$1" = "ipa" ] || exit 1
-REALM="$2"
-USER="$3"
-
-HOST="$(hostname -f)"
-SERVICE="HTTP/${HOST}@${REALM}"
+prefix="@prefix@"
 COCKPIT_GROUP="@COCKPIT_GROUP@"
+COCKPIT_WS_CERTS_D="@sysconfdir@/cockpit/ws-certs.d"
+COCKPIT_RUNTIME_DIR="/run/cockpit"
 
-# use a temporary keytab to avoid interfering with the system one
-export KRB5CCNAME=/run/cockpit/keytab-setup
+install_cert() {
+    local DESTINATION="${COCKPIT_WS_CERTS_D}/$1"
```

Comment:
Style nitpick: global vars should be UPPER_CASE, local vars lower_case IMHO. (opposite problem with `$prefix`)

Reply:
`$prefix` is there to deal with the fact that the expansion for `@sysconfdir@` ends up containing, literally, `${prefix}`, so we don't have a lot of room to fix that.

In one of my attempts to do variable expansion in the manifests from webpack I had some tweaks to configure.ac to set an absolute version of @libexecdir@, and we could do a similar trick here.  Or we could just suffer with it as it is.

I was on the edge of adding a comment, and I think I will now...